composer.json

```diff
@@ -34,7 +34,12 @@
   "autoload": {
     "psr-4": {
       "Localheinz\\OrganizingTestCodeInPhp\\": "src/"
-    }
+    },
+    "exclude-from-classmap": [
+      "/src/*Bench.php",
+      "/src/*Test.php",
+      "/src/Helper.php"
+    ]
```

Comment:
When you declare test code in the same directory as production code, managing exclusions for the `composer` autoloader in `composer.json ` is a lot of work.